requirements.txt

```diff
@@ -1,10 +1,12 @@
-coverage==4.5.1
-psutil==5.6.6
+setuptools~=54.2.0
+
 requests==2.20.0
-responses==0.9.0
 tqdm==4.19.6
 htsget==0.2.5
+psutil==5.6.6
 urllib3==1.24.2
 
 pytest~=6.0.1
-setuptools~=54.2.0
\ No newline at end of file
+coverage==4.5.1
+responses==0.9.0
+pyfakefs~=4.5.1
```

Comment:
I didn't increase or decrease any versions, I only 
* reorganized the dependencies (into 2 groups: non-test and test-dependencies) 
* and I added a new test-dependency (pyfakefs), which provides an in-memory fake file-system and which made my life much easier
** when I had to write tests which create (=download) files
** and when I had to check the existence of those downloaded files.